.github/workflows/publish.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish Release
+
+on:
+  workflow_dispatch:
+  release:
+    types:
+      - published
+
+permissions:
+  contents: read
+
+jobs:
+  publish-npm:
+    name: 'NPM'
+    runs-on: ubuntu-latest
+    environment: release
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'npm'
+
+      - name: Build package
+        run: npm run build
```

Comment:
Is this step required? Would it not be better to handle this in the `prepublishOnly` script?

Reply:
Cool, then let's just leave it as is :)